frontend/src/locales/es.json

```
@@ -26,8 +26,8 @@
     "moderator": "Moderador",
     "moderators": "Moderadores",
     "myContributions": "Mis contribuciones al bien común",
-    "openContributionLinks": "lista de enlaces de contribuciones públicas",
-    "openContributionLinkText": "La comunidad \"{name}\" proporciona actualmente las siguientes {count} creaciones automáticas.",
+    "openContributionLinks": "Creaciones generadas por enlace",
```

Hier fehlt ``"noOpenContributionLinkText": "Actualmente no hay creaciones automáticas.",``
```suggestion
    "noOpenContributionLinkText": "Actualmente no hay creaciones automáticas.",
    "openContributionLinks": "Creaciones generadas por enlace",
```